use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -38,9 +35,181 @@ console.
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
+## Automate tiering with policies
+
+A tiering policy automatically moves data to the object storage tier. Any chunks that only contain data
```

Comment:
A tiering policy automatically moves any  data chunks that only contain data
older than the `move_after` threshold to the object storage tier.